torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -45,6 +46,19 @@ bool IsNonTrivialDilation(at::IntList dilation) {
       [](const int64_t dim_dilation) { return dim_dilation != 1; });
 }
 
+// Consistent with the checks in bmm_out_or_baddbmm_.
+void CheckBmmArguments(const at::Tensor& batch1, const at::Tensor& batch2) {
```

Comment:
Come on. I think we can check ourselves instead of this.
We are using XLA_* all over, we better not add a new thing within our domain.

https://pytorch.org/docs/stable/torch.html#torch.bmm

Reply:
Alright, I'll do it in `XLATensor` instead.

Reply:
We can use XLA_* even in there if you want. That is our code as well :)

Reply:
True, but if we're redoing the dimensional checks anyway, it makes sense to sink them into `XLATensor` and keep `AtenXlaType` trivial.

Reply:
Done.